docs/frontend/plain-js.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Javascript and Initializers
+title: JavaScript and Initializers
 ---
 
-# Javascript and Initializers
+# JavaScript and Initializers
 
-DEV has two Javascript codebases.
+DEV has two JavaScript codebases.
 
 One is located in the directory `app/assets/javascripts` and contains plain
-Javascript (mostly ES5+) being served using
+JavaScript (mostly ES5+) being served using
 [Sprockets](https://github.com/rails/sprockets-rails) which packages static
 assets.
 
-Webpacker manages the other one, and you can read more about [in its own
-guide](/frontend/webpacker).
+Webpacker manages the other one, and you can read more about
+[in its own guide](/frontend/webpacker).
```

Comment:
Read more about “it” in its own guide?

Reply:
Pushed, updated the same verbiage in another place!